src/index.js

```diff
@@ -2,20 +2,10 @@ import React, { Component } from "react";
 import ReactDom from "react-dom";
 import TestUtils from "react-dom/test-utils";
 
-class StatelessWrapper extends Component {
-  render() {
-    return this.props.children; // eslint-disable-line react/prop-types
-  }
-}
-
-export function mount(component) {
-  const rendered =
-    TestUtils.renderIntoDocument(component) ||
-    TestUtils.renderIntoDocument(
-      <StatelessWrapper>{component}</StatelessWrapper>
-    );
-
-  return ReactDom.findDOMNode(rendered);
+export function mount(element) {
+  const div = document.createElement('div');
+  ReactDOM.render(element, div);
+  return div;
```

Comment:
Don't we need to return `div.firstChild`?

Reply:
That makes the tests pass.

Reply:
I'm sorry, but I would really like the mount function to just mount the element and return the mounted node. I think it is a bit confusing getting an extra `div` I didn't ask for. I would like any utility methods like the `queryByTestId` to be exported separately and take a node as the argument. We might have a bit different use-cases as I'm basically planing to use unexpected-dom on top of this library.

Reply:
Gotcha. I'm good with that. When I use this I can wrap it in a `div` myself if I want so that's fine :+1:

Thanks!